### PR TITLE
Update environment creation document [do not merge until MGuoynes confirms changes complete]

### PR DESCRIPTION
**Pull request by mguoynes**

### CODE REVIEW

File: src/content/topics/home/organizing-flags/environments.mdx [hunk 1]
```diff
@@ -27,16 +27,19 @@ Here is a screenshot of the **Switch environment** menu:
...
 
-
 You can manage your environments from the **Account settings** page. Here, you can add new environments to a project, for example, to give each developer on your team their own environment for local testing.
 
 To add a new environment:
 
 1. Navigate to the **Account settings** page.
 2. Click the **Projects** tab. Find the project to which you want to add a new environment.
-3. Click **Create environment**. The "Create an environment" panel appears:
+3. Navigate to the selected project by clicking on the **Project** link.
```
Comment (mmrj) [inline src/content/topics/home/organizing-flags/environments.mdx:36]:
```suggestion
3. Navigate to the selected project by clicking the name of the project.
```

File: src/content/topics/home/organizing-flags/environments.mdx [hunk 2]
```diff
@@ -128,11 +132,12 @@ To reset an SDK or mobile key:
 
 1. Navigate to the the **Account settings** page.
 2. Click the **Projects** tab.
-3. Click the **overflow menu** next to your environment.
-4. Select "Edit environment." An "Edit environment" panel appears.
-5. In the "Keys" section click **Reset SDK key** or **Reset mobile key**.
-6. (Optional) If you are resetting an SDK key, choose how many hours to keep the current SDK key active for, up to 720 hours (30 days).
-6. Enter the name or key of your environment:
+3. In the **Projects** results click on a project name to navigate to that project.
```
Comment (mmrj) [inline src/content/topics/home/organizing-flags/environments.mdx:135]:
```suggestion
3. Navigate to the selected project by clicking the name of the project.
```
If I'm following correctly, this step is the same as what you're doing in environments.mdx/line 36. Is that right? If so, let's use the same language.

File: src/content/topics/home/organizing-flags/environments.mdx [hunk 3]
```diff
@@ -151,3 +156,13 @@ If your site has relatively low traffic (fewer than one request per minute), you
 If the TTL is set to zero minutes, the SDK will not use a local cache, and every call to `variation` will make a remote request to our CDN. You can set your TTL to zero in testing environments to see changes reflected immediately, but we do not recommend a zero minute TTL in production.
 
 In high volume PHP environments, we strongly recommend using our Relay Proxy. To learn more, read [The Relay Proxy](/home/relay-proxy).
+
+To set the TTL:
+
+1. Navigate to the the **Account settings** page.
+2. Click the **Projects** tab.
+3. In the **Projects** results click on a project name to navigate to that project.
```
Comment (mmrj) [inline src/content/topics/home/organizing-flags/environments.mdx:164]:
```suggestion
3. Navigate to the selected project by clicking the name of the project.
```
Same comment as above, I think we can standardize the language here.

File: src/content/topics/home/organizing-flags/environments.mdx [hunk 4]
```diff
@@ -151,3 +156,13 @@ If your site has relatively low traffic (fewer than one request per minute), you
...
+4. Click the **overflow menu** next to your environment.
+5. Select "Edit environment." An "Edit environment" panel appears.
+6. Enter a value in the **TTL input** and once done click **Save Environment**.
```
Comment (mmrj) [inline src/content/topics/home/organizing-flags/environments.mdx:167]:
```suggestion
6. Enter a value in the **TTL input** and click **Save environment**.
```

**Review by mmrj — CHANGES_REQUESTED** [review]
Thanks for putting this together! I added some minor copyedits. 

When would you like this published?

File: src/content/topics/home/organizing-flags/environments.mdx [hunk 5]
```diff
@@ -151,3 +156,13 @@ If your site has relatively low traffic (fewer than one request per minute), you
...
 
 In high volume PHP environments, we strongly recommend using our Relay Proxy. To learn more, read [The Relay Proxy](/home/relay-proxy).
+
+To set the TTL:
+
+1. Navigate to the the **Account settings** page.
+2. Click the **Projects** tab.
+3. In the **Projects** results click on a project name to navigate to that project.
+4. Click the **overflow menu** next to your environment.
+5. Select "Edit environment." An "Edit environment" panel appears.
```
Comment (ember-stevens) [inline src/content/topics/home/organizing-flags/environments.mdx:166]:
```suggestion
5. Select "Edit environment." The "Edit environment" panel appears.
```

File: src/content/topics/home/organizing-flags/environments.mdx [hunk 6]
```diff
@@ -128,11 +132,12 @@ To reset an SDK or mobile key:
...
 1. Navigate to the the **Account settings** page.
 2. Click the **Projects** tab.
-3. Click the **overflow menu** next to your environment.
-4. Select "Edit environment." An "Edit environment" panel appears.
-5. In the "Keys" section click **Reset SDK key** or **Reset mobile key**.
-6. (Optional) If you are resetting an SDK key, choose how many hours to keep the current SDK key active for, up to 720 hours (30 days).
-6. Enter the name or key of your environment:
+3. In the **Projects** results click on a project name to navigate to that project.
+4. Click the **overflow menu** next to your environment.
+5. Select "Edit environment." An "Edit environment" panel appears.
```
Comment (ember-stevens) [inline src/content/topics/home/organizing-flags/environments.mdx:137]:
```suggestion
5. Select "Edit environment." The "Edit environment" panel appears.
```

**Comment by mguoynes** [comment]
work in progress..

> Thanks for putting this together! I added some minor copyedits.
> 
> When would you like this published?

It will be a bit, the project isn't completed yet. Once we finish up, would you like me to reach out?

**Comment by mmrj** [comment]
> work in progress..
> 
> > Thanks for putting this together! I added some minor copyedits.
> > When would you like this published?
> 
> It will be a bit, the project isn't completed yet. Once we finish up, would you like me to reach out?

Yes please! We can merge & publish once the implementation is ready to be released. Thanks so much.

**Pull request closed without merging**